state.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"log"
+
+	_ "github.com/dannypsnl/redux"
```

Comment:
I would avoid using `_`. It's very rarely beneficial in the long term.

Reply:
k